sbt-scalajs-bundler/src/main/scala/scalajsbundler/util/Commands.scala

```diff
@@ -8,6 +8,7 @@ import scala.sys.process.ProcessLogger
 object Commands {
 
   def run(cmd: Seq[String], cwd: File, logger: Logger): Unit = {
+    logger.debug(s"Command: ${cmd.mkString(" ")}")
```

Comment:
This is not strictly part of the change but I found this very useful for debugging